src/common/components/moteinnkalling/GiSvarPaInnkallelse.tsx

```diff
@@ -152,77 +152,83 @@ const GiSvarPaInnkallelse = ({ brevUuid }: Props): ReactElement => {
-            description={begrunnelseDescriptionSmallText}
-            maxLength={300}
-            error={isFieldError(inputFields.begrunnelseAvlysning)}
-            onChange={handleChangeBegrunnelse}
-            value={begrunnelse}
-          />
-        </>
-      )}
-
-      {sendSvarQuery.isError && (
-        <Alert variant="error">{texts.errorMessage}</Alert>
-      )}
-
-      <KnappStyled
-        disabled={sendSvarQuery.isLoading}
-        loading={sendSvarQuery.isLoading}
-        onClick={validateAndSubmit}
-      >
-        {texts.sendSvar}
-      </KnappStyled>
+      <form>
+        {error.length > 0 && (
+          <ErrorSummary heading={texts.feiloppsummeringTittel}>
+            {error.map((error, index) => {
+              return (
+                <ErrorSummary.Item key={index} href={`#${error.inputField}`}>
+                  {error.errorMsg}
+                </ErrorSummary.Item>
+              );
+            })}
+          </ErrorSummary>
+        )}
+
+        <RadioGroup
+          id={inputFields.svarType}
+          legend={texts.svarLegend}
+          error={
+            isFieldError(inputFields.svarType) ? texts.svarRequired : undefined
+          }
+          onChange={handleChangeSvarType}
+        >
+          <Radio value="KOMMER">{texts.svarKommer}</Radio>
+          <Radio value="NYTT_TID_STED">{texts.svarEndring}</Radio>
+          <Radio value="KOMMER_IKKE">{texts.svarAvlysning}</Radio>
+        </RadioGroup>
+
+        {svarType === "NYTT_TID_STED" && (
+          <>
+            <Alert variant="info">
+              <BodyLong>{texts.infoEndring}</BodyLong>
+            </Alert>
+
+            <Textarea
+              id={inputFields.begrunnelseEndring}
+              label={texts.begrunnelseEndringLabel}
+              description={begrunnelseDescriptionSmallText}
+              maxLength={300}
+              error={isFieldError(inputFields.begrunnelseEndring)}
+              onChange={handleChangeBegrunnelse}
+              value={begrunnelse}
+            />
+          </>
+        )}
+
+        {svarType === "KOMMER_IKKE" && (
+          <>
+            <Alert variant="info">
+              <BodyLong>{texts.infoAvlysning}</BodyLong>
+            </Alert>
+            <Textarea
+              id={inputFields.begrunnelseAvlysning}
+              label={texts.begrunnelseAvlysningLabel}
+              description={begrunnelseDescriptionSmallText}
+              maxLength={300}
+              error={isFieldError(inputFields.begrunnelseAvlysning)}
+              onChange={handleChangeBegrunnelse}
+              value={begrunnelse}
+            />
+          </>
+        )}
+
+        {sendSvarQuery.isError && (
+          <Alert variant="error">{texts.errorMessage}</Alert>
+        )}
+
+        <KnappStyled
+          type="submit"
+          disabled={sendSvarQuery.isLoading}
+          loading={sendSvarQuery.isLoading}
+          onClick={(e: React.MouseEvent) => {
```

Comment:
Jeg leste et sted at man heller burde bruke onSubmit i stedet for onClick, ellers får man ikke submittet formen med å bruke enter. Kan jo se om det har noe for seg :D